catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/NitfGmtiTransformer.java

```diff
@@ -108,27 +112,31 @@ private void transformTargetLocation(Metacard metacard) {
                 WKTReader wktReader = new WKTReader(geometryFactory);
                 Geometry geometry = wktReader.read(locationString);
 
-                LOGGER.debug("Setting the metacard attribute [{}, {}]", Core.LOCATION,
+                LOGGER.debug("Setting the metacard attribute [{}, {}]",
+                        Core.LOCATION,
                         geometry.toText());
                 metacard.setAttribute(new AttributeImpl(Core.LOCATION, geometry.toText()));
             }
 
         } catch (ParseException e) {
-            LOGGER.debug(e.getMessage(), e);
+            LOGGER.warn(e.getMessage(), e);
```

Comment:
Was this deliberately reverted to the old `warn` level, or was there a possibility this was merged incorrectly? 

See https://github.com/codice/alliance/commit/ddd398490d8e91c682e5718ca87c251aaa4ef376#diff-35230559684c153d7a3432f20c908f3eR117.